js/index.js

```diff
@@ -8,6 +8,14 @@ var path = require('path');
 var pkgInfo = require('../package.json');
 
 var DEFAULT_IGNORE = ['node_modules'];
+var SOURCEMAPS_OPTIONS = {
+  noSourceMapReference: '--no-sourcemap-reference',
+  stripPrefix: '--strip-prefix',
+  stripCommonPrefix: '--strip-common-prefix',
+  validate: '--validate',
+  urlPrefix: '--url-prefix',
+  ext: '--ext'
+};
```

Comment:
It is still passing the `--rewrite` option by default (see line 76), you might wanna add this here.